client/lib/oauth-store/test/index.js

```diff
@@ -88,16 +87,20 @@ describe( 'oAuthStore', function() {
 		} );
 	} );
 
-	it( 'sets OAuth token when login is correct', function() {
-		var before = global.document;
-
-		global.document = { location: { replace: sinon.stub() } };
+	it( 'sets OAuth token when login is correct', sinon.test( function() {
+		const originalDocument = global.document,
+			replaceSpy = this.spy();
 
-		sinon.stub( oAuthToken, 'setToken' );
+		global.document = {
+			location: {
+				replace: replaceSpy
```

Comment:
Any particular reason this was changed from a stub to spy? The latter would typically be used when wanting to let an original behavior be left intact. Does JSDOM implement these functions for us? Could we simply `this.spy( global.document.location, 'replace' );` instead?